app/views/govuk_publishing_components/components/docs/figure.yml

```diff
@@ -15,29 +15,30 @@ uses_component_wrapper_helper: true
 examples:
   default:
     data:
-      src: 'https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg'
+      src: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg
   figure_with_alt_text:
     data:
-      src: 'https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg'
-      alt: 'An image of the lords chamber'
+      src: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg
+      alt: An image of the lords chamber
   figure_with_caption:
+    description: The image will fill the available space but not exceed the width of the container or its own width. The caption will always be the same width as the image.
     data:
-      src: 'https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg'
-      alt: 'An image of the lords chamber'
-      caption: 'The Lords Chamber'
+      src: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg
+      alt: An image of the lords chamber
+      caption: The Lords Chamber on the 14th of August. A large number of people are in attendance, filling the seats in the lower and upper areas. Several people are wearing hats.
   right_to_left_with_caption:
     data:
-      src: 'https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/31637/s300_Visas_-_Website.jpg'
-      alt: 'تأشيرات'
-      caption: 'تأشيرات'
+      src: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/31637/s300_Visas_-_Website.jpg
+      alt: تأشيرات
+      caption: تأشيرات
     context:
       right_to_left: true
   figure_with_credit:
     data:
-      src: 'https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg'
-      credit: Wallis Crankled
+      src: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg
+      credit: Lewis Crankled
   figure_with_caption_and_credit:
     data:
-      src: 'https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg'
-      caption: The Lords Chamber
-      credit: Wallis Crankled
+      src: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/54374/s630_the_lords_chamber.jpg
+      caption: The Lords Chamber on the 14th of August. A large number of people are in attendance, filling the seats in the lower and upper areas. Several people are wearing hats.
+      credit: Lewis Crankled
```

Comment:
Just curious on why the credit was changed? 😄 

(Looks like the real photographer is Roger Harris https://www.gov.uk/government/news/queens-speech-explainer)